internal/datasource/config/config_writer_test.go

```diff
@@ -197,10 +209,11 @@ func TestDeleteFile(t *testing.T) {
 	}
 }
 
+// This test will be fixed in a followup PR
 func TestRollback(t *testing.T) {
 	ctx := context.Background()
 	tempDir := t.TempDir()
-	tenantID, instanceID := helpers.CreateTestIDs(t)
+	_, instanceID := helpers.CreateTestIDs(t)
```

Comment:
we don't use tenantID anymore, maybe this could create a correlation id instead?